src/test/unit_tests/test_upgrade_scripts.js

```diff
@@ -31,12 +31,13 @@ mocha.describe('test upgrade scripts', async function() {
         this.timeout(600000); // eslint-disable-line no-invalid-this
         await system_store.load();
 
-        const account_info = await rpc_client.account.read_account({ email: EMAIL, });
+        const account_info = await rpc_client.account.read_account({ email: EMAIL });
         s3 = new S3({
             endpoint: coretest.get_http_address(),
             credentials: {
                 accessKeyId: account_info.access_keys[0].access_key.unwrap(),
                 secretAccessKey: account_info.access_keys[0].secret_key.unwrap(),
+                accountId: account_info._id
```

Comment:
I don't think you need to add the `accountId` as a part of the S3 client in the `credentials` part.